app/src/main/java/com/kickstarter/ui/activities/ProjectPageActivity.kt

```diff
@@ -327,23 +327,38 @@ class ProjectPageActivity :
         this.viewModel.outputs.projectPhoto()
             .compose(bindToLifecycle())
             .observeOn(AndroidSchedulers.mainThread())
-            .subscribe { binding.mediaHeaderLayout.inputs.setProjectPhoto(it) }
+            .subscribe { binding.mediaHeader.inputs.setProjectPhoto(it) }
 
         viewModel.outputs.playButtonIsVisible()
             .compose(bindToLifecycle())
             .compose(Transformers.observeForUI())
-            .subscribe { binding.mediaHeaderLayout.inputs.setPlayButtonVisibility(it) }
+            .subscribe { binding.mediaHeader.inputs.setPlayButtonVisibility(it) }
 
-        binding.mediaHeaderLayout.outputs.playButtonClicks()
+        binding.mediaHeader.outputs.playButtonClicks()
             .compose(bindToLifecycle())
             .compose(Transformers.observeForUI())
             .subscribe { viewModel.inputs.playVideoButtonClicked() }
 
         viewModel.outputs.backingViewGroupIsVisible()
             .compose(bindToLifecycle())
             .compose(Transformers.observeForUI())
-            .subscribe { binding.backingGroup.visibility = it.toVisibility() }
+            .subscribe {
+                binding.backingGroup.visibility = it.toVisibility()
+            }
 
+        viewModel.outputs.hideVideoPlayer()
+            .compose(bindToLifecycle())
+            .compose(Transformers.observeForUI())
+            .subscribe {
+                if (it) {
```

Comment:
When hitting overviewTab, the toolbar should be expanded, And that's currently not working 

https://user-images.githubusercontent.com/4083656/142607788-c6aaa50f-2651-4644-a47a-25ff0ea6b2fb.mp4

All the subscribe block here could be replaced by : 
`binding.projectAppBarLayout.setExpanded(!it)`

Reply:
This is a spec change and the new expected behavior as per discussion with @paulomcg yesterday.